docs/DeveloperGuide.md

```diff
@@ -2,14 +2,191 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
+original source as well}
 
 ## Design & implementation
```

Comment:
Perhaps you could include the overall architecture of the application so that first-time developers would be able to have a clearer picture of your application without having to trace through the source code.

[See example](https://se-education.org/addressbook-level3/DeveloperGuide.html#architecture)